dask/tests/test_base.py

```diff
@@ -1293,6 +1300,21 @@ def test_normalize_function_limited_size():
     assert 50 < len(function_cache) < 600
 
 
+def test_normalize_function_dataclass_field_no_repr():
+    from dataclasses import field, make_dataclass
```

Comment:
Nitpick: Could you move these `import`s to the top of this module along with the other `dataclasses` `import`s we already have 

https://github.com/dask/dask/blob/3d0cd48f3448d993dad54fd59955719f208b0d48/dask/tests/test_base.py#L8

Reply:
sounds good!